tools/scripts/pl_batt_obj/compress.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/python3
```

Comment:
wanted to make a note of this, this file exists because nitrogfx LZ compression doesn't match properly, but this compression algorithm does match properly (I pulled this from the github link below, there was one change to adjust padding bytes from 0xFF to 0x00 right at the bottom to get a properly-matching file)

ideally this gets implemented into nitrogfx directly, i played with changing the nitrogfx implementation a bit but didn't solve it. maybe just need an entirely new converter using this algorithm instead.

Reply:
I'll tackle translating this into C to port over to `nitrogfx`; thanks for the find!